M2/Macaulay2/d/expr.d

```diff
@@ -349,6 +349,7 @@ export RRiClass := newbignumbertype();
 export pointerClass := newbasictype();
 export atomicIntClass := newbasictype();
 -- all new types, dictionaries, and classes go just above this line, if possible, so hash codes don't change gratuitously!
+export pseudocodeClosureClass := newtypeof(pseudocodeClass);
```

Comment:
Based on what the comment right above this line says, this line should probably be moved up a line.